usecases/serviceDiscovery.go

```diff
@@ -81,42 +81,48 @@ func ExtractQuestForm(bodyBytes []byte) (rec forms.ServiceQuest_v1, err error) {
 	return
 }
 
+func sendHttpReq(method string, url string, data []byte, ctx context.Context) (resp *http.Response, err error) {
```

Comment:
Might want to put this in a file with http helpers, let it chill for now until we see more uses for it?